client/contracts/truffle/test/oracle/OracleMgr.test.js

```diff
@@ -9,49 +9,188 @@ require('chai')
   .use(require('chai-as-promised'))
   .use(require('chai-bignumber')(web3.BigNumber))
   .should();
-
+const namehash = require('eth-ens-namehash');
+const { EVMError } = require('../helpers/testUtils.js');
 const { Contracts } = require('zos-lib');
 
 const OracleMgr = artifacts.require('OracleMgr.sol');
 const ConsensusMock = artifacts.require('ConsensusMock.sol');
+const DomainResolverMock = artifacts.require('DomainResolverMock.sol');
 const PublicResolver = artifacts.require('PublicResolver.sol');
 const KNS = artifacts.require('KNSRegistry.sol');
 const FIFSRegistrar = artifacts.require('FIFSRegistrar.sol');
 const UpgradeabilityProxy = Contracts.getFromNodeModules('zos-lib', 'UpgradeabilityProxyFactory');
 const AdminUpgradeabilityProxy = Contracts.getFromNodeModules('zos-lib', 'AdminUpgradeabilityProxy');
-const namehash = require('eth-ens-namehash');
-const { EVMError } = require('../helpers/testUtils.js');
 
-contract('Oracle Manager', ([_, admin, owner, newOwner, notOwner]) => {
-  beforeEach(async () => {
-    this.proxyFactory = await UpgradeabilityProxy.new();
-    this.knsContract = await KNS.new({ from: owner });
-    this.logs = await this.proxyFactory.createProxy(admin, this.knsContract.address, { from: admin });
-    this.logs1 = this.logs.logs;
-    this.knsProxyAddress = this.logs1.find(l => l.event === 'ProxyCreated').args.proxy;
-    this.knsProxy = await AdminUpgradeabilityProxy.at(this.knsProxyAddress);
-    this.kns = new KNS(this.knsProxyAddress);
-    await this.kns.initialize(owner);
-    this.registrar = await FIFSRegistrar.new(this.knsProxyAddress, namehash('kowala'));
-    this.resolver = await PublicResolver.new(this.knsProxyAddress);
-
-    await this.kns.setSubnodeOwner(0, web3.sha3('kowala'), this.registrar.address, { from: owner });
-    await this.registrar.register(web3.sha3('validator'), owner, { from: owner });
-    await this.kns.setResolver(namehash('validator.kowala'), this.resolver.address, { from: owner });
-    this.consensus = await ConsensusMock.new(true);
-    await this.resolver.setAddr(namehash('validator.kowala'), this.consensus.address, { from: owner });
-    this.oracle = await OracleMgr.new(1, 1, 1, this.resolver.address, { from: owner });
+contract('OracleMgr', ([_, admin, owner, newOwner, newOwner2, newOwner3, newOwner4, notOwner]) => {
```

Comment:
we're not covering the deploy test - to verify if the initial values (constructor) are the one we expected.

Reply:
Test for that is in the section "Setter and Getters"